Misc/NEWS.d/next/Library/2019-05-19-10-48-46.bpo-21315.PgXVqF.rst

```diff
@@ -0,0 +1,3 @@
+Email headers containing 2047 encoded words with no leading whitespace are
+parsed correctly. Also, missing trailing whitespaces now register a defect
```

Comment:
are parsed despite the missing whitespace, and a defect registered.

Also missing trailing whitespace after encoded words is now registered as a defect.